R/app.R

```diff
@@ -340,8 +339,21 @@ loadSupport <- function(appDir, renv=new.env(parent=globalenv()), globalrenv=glo
   }
 
   helpersDir <- file.path(appDir, "R")
+
+  disabled <- list.files(helpersDir, pattern="_disable_autoload\\.r$", recursive=FALSE, ignore.case=TRUE)
```

Comment:
I think the pattern should start with `^`.

Reply:
Touche! Fixed.